### PR TITLE
Integration, v0.9.9

### PR DESCRIPTION
**Pull request by svenseeberg**

### CODE REVIEW

File: bin/docserv [hunk 1]
```diff
@@ -698,5 +706,8 @@ ch.setFormatter(formatter)
 logger.addHandler(ch)
 
 if __name__ == "__main__":
-    docserv = Docserv(sys.argv)
-    docserv.start()
+    if "--help" in sys.argv:
```
Comment (tomschr) [inline bin/docserv]: Maybe allow this:

```python
if "--help" in sys.argv or "-h" in sys.argv:
```

File: bin/docserv [hunk 2]
```diff
@@ -687,6 +687,14 @@ class ThreadedRESTServer(ThreadingMixIn, HTTPServer):
         logger.info("Starting HTTP server on %s:%i" % server_address)
         self.docserv = docserv
 
+def print_help():
+    print("""This is a deamon and should be invoked with the command:
+
+> systemctl start docserv CONFIG_FILENAME
```
Comment (tomschr) [inline bin/docserv]: I'm not an expert in systemd, but it looks a bit unusual to me. Should configuration files reside under `/etc/sysconfig/`?

File: bin/docserv [hunk 3]
```diff
@@ -698,5 +706,8 @@ ch.setFormatter(formatter)
 logger.addHandler(ch)
 
 if __name__ == "__main__":
-    docserv = Docserv(sys.argv)
-    docserv.start()
+    if "--help" in sys.argv:
+        print_help()
+    else:
+        docserv = Docserv(sys.argv)
+        docserv.start()
```
Comment (tomschr) [inline bin/docserv]: Should we return a return code?

File: bin/docserv (hunk 2 above)
Comment (svenseeberg) [reply bin/docserv]: No, configuration files for daemons usually reside in /etc/[PROGRAM]/[CONFIG], for example /etc/apache2/httpd.conf, /etc/postfix/main.cf, etc.

File: bin/docserv (hunk 3 above)
Comment (svenseeberg) [reply bin/docserv]: The script already implicitly returned the correct code (0). Now I added a line to explicitly return 0.

File: bin/docserv (hunk 2 above)
Comment (svenseeberg) [reply bin/docserv]: And passing a parameters, for example config file name with @ is common practice.

Comment (svenseeberg) [reply bin/docserv]: No, I forgot to write it.

**Pull request merged**